portal/apps/org.wso2.carbon.iam.admin-portal.app/src/main/pages/recovery/forced-password-reset.js

```diff
@@ -0,0 +1,8 @@
+
+function onGet(env) {
+    var session = getSession();
+    if (!session || !session.getUser()) {
```

Comment:
This is unnecessary.. Use {{secured}} handlebar in your hbs file to redirect unauthorized users to the login page.